tests/everest/test_everserver.py

```diff
@@ -271,3 +281,46 @@ def test_everserver_status_max_batch_num(
         filter_out_gradient=False, batches=None
     )
     assert {data.batch for data in snapshot.simulation_data} == {0}
+
+
+@patch("sys.argv", ["name", "--config-file", "config_minimal.yml"])
+@patch("everest.detached.jobs.everserver._configure_loggers")
+@patch("everest.detached.jobs.everserver._generate_authentication")
+@patch(
+    "everest.detached.jobs.everserver._generate_certificate",
+    return_value=(None, None, None),
+)
+@patch(
+    "everest.detached.jobs.everserver._find_open_port",
+    return_value=42,
+)
+@patch("everest.detached.jobs.everserver._write_hostfile")
+@patch("everest.detached.jobs.everserver._everserver_thread")
+def test_everserver_status_contains_max_runtime_failure(
+    _1, _2, _3, _4, _5, _6, copy_math_func_test_data_to_tmp
```

Comment:
Do you need to use the math_func test data here? Could create a minimal case with just a sleep job (and can just use `sleep` from the command line and not have to add to the existing script.

There is an existing fixture for this:
```python
@pytest.fixture
def min_config():
    yield yaml.safe_load(
        dedent("""
    model: {"realizations": [0]}
    controls:
      -
        name: my_control
        type: well_control
        min: 0
        max: 0.1
        variables:
          - { name: test, initial_guess: 0.1 }
    objective_functions:
      - {name: my_objective}
    config_path: .
    """)
    )
```